README.md

```diff
@@ -27,11 +29,9 @@ helm repo update
 ```
 
 For installation of each integration, please go inside each intergation's directory:
-- [Fluentd-Coralogix chart](https://github.com/coralogix/eng-integrations/blob/master/fluentd/coralogix/README.md)
-- [Fluentd-HTTP chart](https://github.com/coralogix/eng-integrations/blob/master/fluentd/http/README.md)
-- [Fluent-bit-Coralogix chart](https://github.com/coralogix/eng-integrations/blob/master/fluent-bit/coralogix/README.md)
-- [Fluent-bit-HTTP chart](https://github.com/coralogix/eng-integrations/blob/master/fluent-bit/http/README.md)
-- [OpenTelementry-Agent chart](https://github.com/coralogix/eng-integrations/blob/master/otel-agent/README.md)
+- [Fluentd-HTTP chart](https://github.com/coralogix/telemetry-shippers/blob/master/logs/fluentd/http/README.md)
+- [Fluent-bit-HTTP chart](https://github.com/coralogix/telemetry-shippers/blob/master/logs/fluent-bit/http/README.md)
+- [OpenTelementry-Agent chart](https://github.com/coralogix/telemetry-shippers/blob/repo-redesign/otel-agent/README.md)
```

Comment:
what about fluentbit-coralogix and the prometheus ecs ?

Reply:
i removed them because i dont want people to use them, i will keep them and add a DEPRECATED on the READme